plugins/permission-react/src/apis/IdentityPermissionApi.ts

```diff
@@ -47,8 +48,21 @@ export class IdentityPermissionApi implements PermissionApi {
   async authorize(
     request: EvaluatePermissionRequest,
   ): Promise<EvaluatePermissionResponse> {
+    const { permission, resourceRef } = request;
+    if (isResourcePermission(permission)) {
```

Comment:
We can enforce this by setting `request: AuthorizePermissionRequest`! This implementation and the corresponding `PermissionApi` interface don't have a `query()` method though. I think that's okay considering the that's more of a backend use case